sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 12.0.0-beta.8 (Unreleased)
```

Comment:
Nice, it works. Though I am surprised, I thought it validates the release date of the previous version (format YYYY-MM-dd), and we have `Unreleased`. I am guessing the validation is part of a different step.